FAQ.md

```diff
@@ -381,3 +382,11 @@ it can [cause
 problems](https://groups.google.com/d/msg/golang-nuts/AnMr9NL6dtc/UnyUUKcMCAAJ).
 If your dependers are using `dep`, this is not a concern, as `dep` takes care of
 stripping out nested `vendor` directories.
+
+## One or more of my dependencies doesn't tag its releases. What should I do?
+
+Simply add a constraint to your manifest that specifies `"branch": "master"`
```

Comment:
Remove "simply", changing it to "Add a constraint...". I don't want anyone feeling bad about themselves if this doesn't feel simple! 😀